packages/ui/src/components/smui-wrapper/custom-drawer/custom-drawer.store.ts

```diff
@@ -0,0 +1,61 @@
+// SVELTE
+import { writable, get } from 'svelte/store'
+// TYPES
+import type { DrawerPayload } from './custom-drawer.types'
+
+//==== STATE
+const open = writable<boolean>(false)
+
+const currentTitle = writable<DrawerPayload['title']>('')
+const currentDescription = writable<DrawerPayload['description']>('')
+const currentInnerComponent = writable<DrawerPayload['component']>(undefined)
+const currentInnerComponentProps =
+	writable<DrawerPayload['componentProps']>(undefined)
+
+//==== ACTIONS
+
+function openDrawer({
```

Comment:
Then the `openDrawer`, `closeDrawer`, `switchDrawer` functions can be refactored to use this combined store object.

Reply:
Refactored see 584f1fc